_data/fellow-specialties.yml

```diff
@@ -0,0 +1,4 @@
+- data
+- frontend development
+- backend development
+- product
```

Comment:
@keithpif This is the file y'all will need to put the fellows' specialties in. You'll probably want to capitalize them all nice, unlike what I did here.

Reply:
> @keithpif I believe that has something to do with how the Jekyll paginator element is set up. I'm out the rest of today but happy to look into this later if you like.

yup! I see it in the config file. I'll change to 12